.env.example

```diff
@@ -0,0 +1,27 @@
+# django
+APP_ENV='local' # local, staging, production
+SECRET_KEY= # generate from https://djecrety.ir/
+DEBUG=False
```

Comment:
by default should this be True?

Reply:
I think it depends on the user. having a different variable other than APP_ENV gives the flexibility to have any setting in place.

We can of course add the logic to make it default to True if app_env is local in settings.py. But I see that's more or less redundant.